app/models/chore.rb

```diff
@@ -18,4 +35,14 @@ def due_class(due_date = next_due)
       'due-later'
     end
   end
+
+  def add_business_days(start_date:, work_days:)
+    raise ArgumentError, 'start_date must be a Date' unless start_date.instance_of?(Date)
+    raise ArgumentError, 'work_days must be a positive integer' unless work_days.is_a?(Integer) && work_days > 0
+    start_date += 1 if start_date.wday == 6
```

Comment:
`wday` is a method for the `Date` class that returns the weekday of a given date:
https://ruby-doc.org/stdlib-2.3.1/libdoc/date/rdoc/Date.html#method-i-wday

`divmod` is a method for the `Numeric` class that returns the quotient and remainder as an array:
https://ruby-doc.org/core-2.3.0/Numeric.html#method-i-divmod